app/css/lab-orders-list.scss

```diff
@@ -46,3 +46,7 @@
     max-width: 90px;
 }
 
+.rt-tr-group {
```

Comment:
I'm not really sure what this styling does, please add a comment on top because it's a bit ambiguous.